packages/pyright-internal/src/tests/completions.test.ts

```diff
@@ -1631,11 +1642,11 @@ describe('deprecated', () => {
         const code = `
 // @filename: test.py
 //// from warnings import deprecated
-//// 
-//// 
+////
+////
 //// @deprecated('asdf')
 //// class Asdf: ...
-//// 
+////
```

Comment:
These whitespace changes should be reverted. Assuming they were made accidentally by your editor, they have the potential to cause more conflicts than necessary when merging upstream

Reply:
Sorry, haven't noticed them. Reverted, let's wait for this CI round to complete, and I'll push the polishing commits right after.